### ISSUE TITLE
Consider dependencies when adding classes to the workspace.

### ISSUE THREAD

**Issue by leon-k-martin** [the-virtual-brain/tvb-ext-ontology#4]
## Background
Some Methods have specific requirements/dependencies. For example, the `JansenRit` model requires its own coupling `SigmoidalJansenRit`

## Goal
- When the user adds a class to the workspace, it should automatically load the dependencies.
- Requirements should also be listed in the Panel next to Parents and Children.

## Implementation
I added the required nodes already to the class dictionary:

```python
def ontoclass2dict(ontoclass):
    """
    Convert an OntoClass object to a JSON object
    Args:
        ontoclass: OntoClass object
    Returns:
        dict: JSON object
    """
    return {
        "id": uid2int(ontoclass.identifier),
        "iri": ontoclass.iri,
        "label": ontoclass.label.first(),
        "symbol": label2symbol(ontoclass),
        "type": ontology.get_type(ontoclass).label.first(),
        "definition": ontoclass.definition.first(),
        "collapsed": True,
        "childNodes": [],
        "childLinks": [],
        "is_a": [p.name for p in ontoclass.is_a if isinstance(p, owl.ThingClass)],
        "requires": [
            p.name for p in ontoclass.requires
        ],
    }
```

Would it be possible to implement a mechanism in the front-end that co-adds the corresponding classes to the workspace?

**Comment by romina1601** [comment]
Yes, we can totally do this. What would be the strategy in case there is more than just one option available?

**Comment by leon-k-martin** [comment]
Maybe we can highlight the edges representing dependencies so the user can choose between options if more are available. Something like in https://github.com/vasturiano/react-force-graph/blob/master/example/highlight/index.html

<img width="806" alt="image" src="https://github.com/user-attachments/assets/5b2fecf9-f28c-41e4-a7ea-18d811410527">

What do you say?
